x/fundraising/client/cli/tx.go

```diff
@@ -236,9 +236,13 @@ in order to bid for the amount of coin you bid for the auction.
 				return err
 			}
 
+			// TODO: receive bid type argument
```

Comment:
Checked to do after this is merged.
I think MsgPlaceBid part can be modified according to the msg state.

Reply:
Yes, CLI command for `MsgPlaceBid` needs to be changed due to the updated spec design. Since this PR is about missing args and rpc message, I left that TODO comment and created an issue https://github.com/tendermint/fundraising/issues/94. We can handle it when we are done with msg server and gRPC logics.